vision/unit_tests/test_client.py

```diff
@@ -81,6 +81,51 @@ def test_image_with_client(self):
         image = client.image(source_uri=IMAGE_SOURCE)
         self.assertIsInstance(image, Image)
 
+    def test_multiple_detection_from_content(self):
+        from google.cloud.vision.feature import Feature
+        from google.cloud.vision.feature import FeatureTypes
+        from unit_tests._fixtures import LABEL_DETECTION_RESPONSE
+        from unit_tests._fixtures import LOGO_DETECTION_RESPONSE
+        RETURNED = LABEL_DETECTION_RESPONSE
```

Comment:
1. `returned` isn't a constant (I realize we have violaters elsewhere, no need to pile on)
1. Empty line between imports and code